app/controllers/pages_controller.rb

```diff
@@ -4,5 +4,6 @@ def home
     @form = BalanceTransactionForm.new(Transaction.new, balance_transactions: BalanceTransaction.new)
     @transactions = current_user.transactions.includes(:transactinable).limit(10).order('id desc')
     @categories = current_user.categories.main_category
+    @date = Charts::Home::CategoriesDate.new(current_user.categories_have_budget).date
```

Comment:
@marderer maybe you were thinking about `data`? 😄

Reply:
@AleksSenkou Maybe =)